pkg/routeagent-driver/handlers/kubeproxy-iptables/kp-iptables.go

```diff
@@ -0,0 +1,100 @@
+package kp_iptables
+
+import (
+	"k8s.io/klog"
+
+	"github.com/submariner-io/admiral/pkg/log"
+	k8sV1 "k8s.io/api/core/v1"
+
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/event"
+)
+
+type SyncHandler struct {
+	event.HandlerBase
+	clusterID        string
+	localClusterCidr []string
+	localServiceCidr []string
+
+	cniIface *cniInterface
+}
+
+func NewSyncHandler() *SyncHandler {
+	return &SyncHandler{}
+}
+
+func (kp *SyncHandler) GetName() string {
+	return "kubeproxy-iptables-handler"
+}
+
+func (kp *SyncHandler) GetNetworkPlugin() string {
+	return event.AnyNetworkPlugin
```

Comment:
In our case it will be "generic" or "Generic" whatever subctl reports on network discovery.

Reply:
Okay, I'll update it.

Reply:
Well, now that I think it can be generic or OpenshiftSDN, probably when the generic is ready we can make  a sub-class driver, which only changes the name...

In that way if we had any openshift SDN specifics we could handle them there :)

Reply:
Let's do this on a separate PR when we start having more than one.